sc-memory/sc-memory/sc_agent_manager.hpp

```diff
@@ -335,12 +335,15 @@ class _SC_EXTERN ScAgentManager : public ScObject
    * It determines if the agent class is associated with the specific event that indicates an element
    * is about to be erased.
    *
+   * @param context A sc-memory context used to check for subscription to the erase event.
    * @param agentClassName The name of the agent class to check for subscription to the erase event.
    *
    * @return true if the specified agent class is subscribed to the event of an element being erased;
    *         false otherwise.
    */
-  static _SC_EXTERN bool IsAgentSubscribedToEventOfErasingElement(std::string const & agentClassName);
+  static _SC_EXTERN bool IsAgentSubscribedToEventOfErasingElement(
```

Comment:
can you add something about element that no longer exists being erased to method name and docs?

Reply:
Fixed